### DISCUSSION TITLE
Using the "tab" ident to format my notes doesn't carry over to mkdocs

### DISCUSSION THREAD

**Discussion by Crelemental** [squidfunk/mkdocs-material#6990] [Q&A]
So for my notes, rather than using * to create nested indents, I just use the Tab key to indent the text. It looks something like this in Obsidian, the markdown editor I use.
![image](https://github.com/squidfunk/mkdocs-material/assets/28940622/ed861ccf-d4c1-426c-8e89-0f96f193acd8)
and like this in notepad
![image](https://github.com/squidfunk/mkdocs-material/assets/28940622/6caf89e2-553b-4c62-b8e1-ad24872a56e8)
On the website I have published using https://github.com/jobindjohn/obsidian-publish-mkdocs, which uses mkdocs-material, it doesn't read those indents and just makes it a flat text
![image](https://github.com/squidfunk/mkdocs-material/assets/28940622/dddb4264-1c70-4aa3-b28f-afe8b59ac38e)

I searched for a while to see if anyone else had this issue, or if it's not possible, so thought I'd ask here. Most solutions pertain to nested lists. Thanks for any help and advice in advance

**Comment by alexvoss** [comment]
You can [get line breaks by adding two spaces at the end of a line](https://daringfireball.net/projects/markdown/syntax#p). Better tell your editor not to remove these when saving when you are editing a Markdown file!

As for the tabs leading to an indent, I am not sure. Obsidian does seem to be doing its own thing both in respect to line breaks and tabs. There might be a plugin that converts this?
